docs/guides/INDUSTRY-MAPPING-ANALYSIS.md

```
@@ -0,0 +1,403 @@
+   Industry.objects.update_or_create(
+       induty_code=cat.representative_kis_id,
+       defaults={
+           "name": cat.name,
+           "description": f"{cat.name} 지수 (KSIC:{cat.ksic_code} 매핑 결과)"
+       }
+   )
+   ```
+
+## 매핑 흐름도
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)